packages/mds-utils/index.ts

```diff
@@ -554,7 +554,7 @@ function isStateTransitionValid(eventA: VehicleEvent, eventB: VehicleEvent) {
       }
     case VEHICLE_STATUSES.inactive:
       switch (eventB.event_type) {
-        case VEHICLE_EVENTS.register: // FIXME not totally certain about this one..? Check w/ max
+        case VEHICLE_EVENTS.register:
```

Comment:
Still gotta check this! @karcass What do you think, do `register` events match to an `inactive` status?

Reply:
no, they should go to `removed`